warehouse/integrations/vulnerabilities/utils.py

```diff
@@ -89,6 +89,8 @@ def _analyze_vulnerability(request, vulnerability_report, origin, metrics):
             source=origin,
             link=report.advisory_link,
             aliases=report.aliases,
+            details=report.details,
```

Comment:
will these be backfilled by some mechanism or are they new from the integrator?

Reply:
Since these aren't currently being used for anything, we will 'backfill' by wiping the existing rows and re-ingesting all the reports.